### PR TITLE
added forced bucketing routes

### PR DESCRIPTION
**Pull request by caitlinrubin-optimizely**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -94,6 +94,142 @@ server.route({
...
+      result_3: null,
+      result_4: null,
+      user_profiles: userProfileService ? userProfileService.getProfiles() : null,
+    }
+
+    const didSetForcedVariation1 = optly.setForcedVariation(experimentKey1, userId1, forcedVariationKey1);
+    if (!didSetForcedVariation1) { reply(response).code(400) };
+    const didSetForcedVariation2 = optly.setForcedVariation(experimentKey2, userId1, forcedVariationKey2);
+    if (!didSetForcedVariation2) { reply(response).code(400) };
+    const didSetForcedVariation3 = optly.setForcedVariation(experimentKey1, userId2, forcedVariationKey1);
```
Comment (alda-optimizely) [inline main.js]: `experimentKey3`

File: main.js [hunk 2]
```diff
@@ -94,6 +94,142 @@ server.route({
...
+    if (!didSetForcedVariation3) { reply(response).code(400) };
+    const didSetForcedVariation4 = optly.setForcedVariation(experimentKey2, userId2, forcedVariationKey2);
```
Comment (alda-optimizely) [inline main.js]: `experimentKey4`

File: main.js [hunk 3]
```diff
@@ -94,6 +94,142 @@ server.route({
...
+    if (!didSetForcedVariation4) { reply(response).code(400) };
+
+    const variation1 = optly.getForcedVariation(experimentKey1, userId1);
+    const variation2 = optly.getForcedVariation(experimentKey2, userId1);
+    const variation3 = optly.getForcedVariation(experimentKey1, userId2);
```
Comment (alda-optimizely) [inline main.js]: `experimentKey3`

File: main.js [hunk 4]
```diff
@@ -94,6 +94,142 @@ server.route({
...
+    const variation4 = optly.getForcedVariation(experimentKey2, userId2);
```
Comment (alda-optimizely) [inline main.js]: `experimentKey4`

File: main.js (hunk 1 above)
Comment (caitlinrubin-optimizely) [inline main.js]: I dont think experimentKey3 exists?

Comment (alda-optimizely) [inline main.js]: my bad...ignore!

**Pull request merged**